tests/cli/test_cli_regression_mol+mol.py

```diff
@@ -0,0 +1,34 @@
+"""This tests the CLI functionality of training and predicting a regression model on a multi-molecule.
+"""
+
+import pytest
+
+from chemprop.cli.main import main
+
+pytestmark = pytest.mark.CLI
+
+
+@pytest.fixture
+def data_path(data_dir):
+    return str(data_dir / "regression" / "mol+mol.csv")
+
+
+@pytest.fixture
+def checkpoint_path(data_dir):
+    return str(data_dir / "example_model_v2_regression_multi.ckpt")
+
+
+def test_training(monkeypatch, data_path):
+    args = ["chemprop", "train", "-i", data_path, "--smiles-columns", "smiles", "solvent", "--epochs", "1", "--num-workers", "4"]
```

Comment:
we can't really assume we'll have access to more than 1 CPU
```suggestion
    args = ["chemprop", "train", "-i", data_path, "--smiles-columns", "smiles", "solvent", "--epochs", "1", "--num-workers", "0"]
```